src/project-css/app/Models/Table.php

```diff
@@ -67,6 +68,37 @@ public function getRecord($id) {
                     ->get();
     }
 
+    // Return field type for column name passed
+    public function getColumnType($name) {
+        // Get Description of table
+        $columns = DB::select("DESCRIBE `{$this->tblNme}`");
+        // Return Type for Field
+        foreach ($columns as $col) {            
```

Comment:
There is too much logic in here, and again duplicate information where you are setting column types for data.  This should point to a shared library the handles all logic for you, it may be difficult to figure out that abstraction, but it needs to happen.

Reply:
Updated Table model consolidated duplication and broke down function into separate single responsibility functions.